ios/RNSentry.mm

```diff
@@ -207,6 +207,19 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
         }
     }];
 
+    NSDictionary<NSString *, id> *extraContext = [PrivateSentrySDKOnly getExtraContext];
```

Comment:
Is that a new method? we don't have that on Flutter, wondering if we should.

Reply:
Yes, it's a new method, [added here](https://github.com/getsentry/sentry-cocoa/pull/2855/files#diff-127399cc36e15a55aa8f32df346b6407d4e9faab0cf65b94188cbd8ba2c9bbed).

I believe flutter should have it too.